src/main/java/net/clgd/ccemux/rendering/awt/AWTRenderer.java

```diff
@@ -289,7 +285,7 @@ private boolean allowKeyEvents() {
 
 	@Override
 	public void keyTyped(KeyEvent e) {
-		if (isPrintableChar(e.getKeyChar()) & allowKeyEvents()) {
+		if (Utils.isPrintableChar(e.getKeyChar()) & allowKeyEvents()) {
```

Comment:
This should be `&&`, no?

Reply:
Yeah, my bad

Reply:
Actually technically that's the fault of whoever wrote that line, all I did was refactor `isPrintableChar`, but I'll fix it anyways